lib/module-attach/api/lineModuleAttachClient.ts

```diff
@@ -86,17 +86,23 @@ export class LineModuleAttachClient {
     brandType?: string,
   ): Promise<AttachModuleResponse> {
     const formParams = {
-      grantType: grantType,
+      grant_type: grantType,
       code: code,
-      redirectUri: redirectUri,
-      codeVerifier: codeVerifier,
-      clientId: clientId,
-      clientSecret: clientSecret,
+      redirect_uri: redirectUri,
+      code_verifier: codeVerifier,
+      client_id: clientId,
+      client_secret: clientSecret,
       region: region,
-      basicSearchId: basicSearchId,
+      basic_search_id: basicSearchId,
       scope: scope,
-      brandType: brandType,
+      brand_type: brandType,
```

Comment:
It must be snake case. https://github.com/line/line-openapi/blob/bba16e1ac5dc10fa5b73c62f957c1005061ab096/module-attach.yml#L39-L69

Reply:
Woot

Reply:
We discussed this internally, I'll submit change as other PR.